### PR TITLE
Caroline/dictionary

### PR DESCRIPTION
**Pull request by cdebbaruah**
Completed dictionary styling for desktop and mobile screens!

### CODE REVIEW

File: src/Pages/Dictionary.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
+/* eslint-disable max-len */
```
Comment (DiyaBaliga) [inline src/Pages/Dictionary.js:1]: THIS IS SO PRETTY OMG just have to get this off my chest first

File: src/Pages/DictionaryPage.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.DictionaryBackground {
```
Comment (DiyaBaliga) [inline src/Pages/DictionaryPage.css]: lowercase + dashes

File: src/Components/Dictionary/DictionaryCard.js [hunk 3]
```diff
@@ -1,65 +1,180 @@
...
         </div>
       </CardContent>
       <div style={{ padding: 10 }}>
-        <CardActions style={{ justifyContent: 'left', marginLeft: 75 }}>
+        <CardActions style={{ justifyContent: 'left', marginLeft: '15%' }}>
           <div>
             { linksArray.length !== 0 ? (
               <Grid
-                style={{ paddingLeft: 15 }}
                 sx={{
```
Comment (DiyaBaliga) [inline src/Components/Dictionary/DictionaryCard.js:159]: in-line styles

**Review by DiyaBaliga — CHANGES_REQUESTED** [review]
CAROLINEEE!!! IT'S SO PRETTY OMG! I'm so impressed. I changed the margins so that the cards line up with the top text (check the most recent commits if you'd like to see how) but otherwise it's beautiful!!

There are a couple remaining in-line styles, so if you could go through and take them out that'd b v helpful! Otherwise it's perfect

**Comment by cdebbaruah** [comment]
> CAROLINEEE!!! IT'S SO PRETTY OMG! I'm so impressed. I changed the margins so that the cards line up with the top text (check the most recent commits if you'd like to see how) but otherwise it's beautiful!!
> 
> There are a couple remaining in-line styles, so if you could go through and take them out that'd b v helpful! Otherwise it's perfect



> CAROLINEEE!!! IT'S SO PRETTY OMG! I'm so impressed. I changed the margins so that the cards line up with the top text (check the most recent commits if you'd like to see how) but otherwise it's beautiful!!
> 
> There are a couple remaining in-line styles, so if you could go through and take them out that'd b v helpful! Otherwise it's perfect

TY for reviewing my PR!! I've removed the in-line styles & renamed class attributes to be lowercase with dashes

**Pull request merged**